src/main.ts

```diff
@@ -35,10 +38,56 @@ app.on("ready", () =>
       // dialog.showMessageBox({ message: process.argv.join("\n"), buttons: ["OK"] })
    }
 
-   const selectedFolder = dialog.showOpenDialogSync({properties: ["openDirectory"]}) ?? []
-
-   if (selectedFolder.length === 0) {
-      app.quit()
+   // TODO: If there is not a saved folder path or it does not exist, then show the dialog to allow the user to select it...   
+   const VALID_IMAGE_EXTENSIONS = [".jpg", ".jpeg", ".png", ".gif", ".bmp", ".webp"]

```

Comment:
Like we discussed, this should match the file extensions the screensaver actually loads in preload.ts